### PR TITLE
Add prefix

### PR DESCRIPTION
**Pull request by alex2844**
```
fastify.register(require('../index'), {
  engine: { ejs: ejs },
  ....,
  propertyName: 'mobile'
})
fastify.register(require('../index'), {
  engine: { ejs: ejs },
  ....,
  propertyName: 'desktop'
})

reply.mobile('index-for-layout.ejs', { text })
reply.desktop('index-for-layout.ejs', { text })
```

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — CHANGES_REQUESTED** [review]
I would not use `prefix` as a name for this option as it's a common option in fastify plugins to identify a route prefix, e.g. `/v1`. It's best to use a different name to avoid conflicts.

Maybe `propertyName`, `name` or `exposeAs`.

**Comment by alex2844** [comment]
I'll be back in a few hours and fix the conflict with the current version

**Review by mcollina — CHANGES_REQUESTED** [review]
could you please document this in the readme?

**Review by mcollina — APPROVED** [review]
lgtm

**Review by zekth — APPROVED** [review]
lgtm but what about showing the example you described in the issue?
Like having 2 different declarations with different propertynames calling different partials?
```js
fastify.register(require('../index'), {
  engine: { ejs: ejs },
  layout: './templates/layout-mobile.hbs'
  propertyName: 'mobile'
})
fastify.register(require('../index'), {
  engine: { ejs: ejs },
  layout: './templates/layout-desktop.hbs'
  propertyName: 'desktop'
})
```
I think this is a revelant example to showcase this feature

File: README.md [hunk 1]
```diff
@@ -76,6 +76,20 @@ fastify.view('/templates/index.ejs', { text: 'text' }, (err, html) => {
 })
 ```
 
+Like having 2 different declarations with different propertynames calling different partials?
```
Comment (zekth) [inline README.md:79]:
```suggestion
Like having 2 different declarations with different propertynames calling different partials:
```

**Review by zekth — CHANGES_REQUESTED** [review]

**Review by lilpoozie2005 — APPROVED** [review]

**Pull request merged**